packages/wallet/src/redux/protocols/concluding/states.ts

```diff
@@ -37,6 +47,16 @@ export interface AcknowledgeChannelConcluded {
   processId: string;
 }
 
+export interface AcknowledgeChannelDoesntExist {
+  type: 'AcknowledgeChannelDoesntExist';
+  processId: string;
+}
+
+export interface AcknowledgeDefundFailed {
+  type: 'AcknowledgeDefundFailed';
+  processId: string;
+}
```

Comment:
Another option here would be to have:
```ts
export interface AcknolwedgeFailure {
    type: 'AcknowledgeFailure',
    processId: string,
    failure: FailureReason,
}
```
It really depends how similarly you treat these states throughout the protocol.

Reply:
👍 I may make this optimisation in a future PR.